### ISSUE TITLE
dyld: Library not loaded - iOS 13.3.1 and embedded Mapbox.framework

### ISSUE THREAD

**Issue by julianrex** [mapbox/mapbox-gl-native-ios#157]
We are tracking an issue with iOS 13.3.1, that results in the following error in the Xcode console, when running on an iOS device:

```
dyld: Library not loaded: @rpath/Mapbox.framework/Mapbox [...] Reason: no suitable image found
```

**This does not affect apps already on the App Store or on your users' devices.**

Please ensure that you are enrolled in a paid developer account. If using a business account, double check you are not using the team annotated with `(Personal Team)`.

Alternatively, you can build-and-run with an iOS simulator, until Apple releases an update.

This is being reported elsewhere:

- https://forums.developer.apple.com/message/404653
- https://github.com/Alamofire/Alamofire/issues/3051
- https://github.com/flutter/flutter/issues/49504

For Cocoapods users: you may have read that remove `use_frameworks!` will help - in our case this won't as we do not (currently) provide a static framework or support building from source; the issue is with embedded dynamic frameworks.

Reported as `FB7563776`.

**Comment by julianrex** [comment]
This appears to happen for the recent iOS 13.4 beta.

**Comment by Fatehkhann** [comment]
I can't get it working. It's happening with all CocoaPods libs.

**Comment by SiarheiFedartsou** [comment]
It seems to be a new Apple restriction: https://stackoverflow.com/questions/60096258/library-not-loaded-rpath-fblpromises-framework-fblpromises-ios-13-3-1

**Comment by jamesburriss** [comment]
I am a paid apple developer and still having this issue?

**Comment by zhuansun0502** [comment]
OK I have struggled a while and I found out 2 ways of fixing this.
1. Sign up for paid Apple Developer account($119CA/year). Some people say it doesnt work. Here's the answer:
RENEW YOUR CERTIFICATE!
Go to your keychain access, generate a new certificate request then follow the instructions on your paid account page to download a new certificate. When it gets renewed, you are good to go.
@jamesburriss

**Comment by zhuansun0502** [comment]
2.  use_modular_headers!
If you have done enough research, you know what I am talking about. The reason I am not using it is because I hate having tons of warning of "not able to update" and "unable to render" sort of things in my  Xcode.
Let me know if this helped

**Comment by jamesburriss** [comment]
The issue was with my certificates.

Thank you very much I was new to all rbis 


James

**Comment by julianrex** [comment]
Noting that our Feedback to Apple was updated with 

> Recent Similar Reports: More than 10
> Resolution: Potential fix identified - For a future OS update.

We will close this issue once we can prove it has been fixed in by an OS update.

**Comment by julianrex** [comment]
Closing as solved in iOS 13.4 (as noted elsewhere).
